sql/updates/world/master/2025_03_11_00_Darkshore.sql

```diff
@@ -0,0 +1,531 @@
+-- quest_template_addon
+
+INSERT INTO `quest_template_addon` (`ID`, `PrevQuestID`, `NextQuestId`, `ExclusiveGroup`)
+VALUES (13583, 13582, 13586, -13581)
+AS new_data
+ON DUPLICATE KEY UPDATE
+    `PrevQuestID` = new_data.`PrevQuestID`,
+    `NextQuestId` = new_data.`NextQuestId`,
+    `ExclusiveGroup` = new_data.`ExclusiveGroup`;
```

Comment:
Our standards are DELETE + INSERT. I cant say if the stuff you did work for all of our supported Versions so we stick to DELETE + INSERT.